README.md

```diff
@@ -338,30 +370,26 @@ One of these keys is called `FEATURES` and it controls a number of features that
 * github
 * no_crawl
 
-Deprecated Features
+## Deprecated Features
```

Comment:
CLA stuff is entirely gone at this point so this whole section should get deleted

Reply:
done